### PR TITLE
a11y for checkboxes on the sharing page issue #5126

### PR DESCRIPTION
**Pull request by Wagner3UB**
Fixing the a11y issue [#5126](https://github.com/plone/volto/issues/5126)

### CODE REVIEW

File: src/components/manage/Sharing/Sharing.jsx [hunk 1]
```diff
@@ -412,8 +416,21 @@ class SharingComponent extends Component {
...
+                              )}
+                              aria-label={this.props.intl.formatMessage(
+                                messages.assignNewRoles,
+                                {
+                                  entry: entry.title,
+                                  role: role.id,
+                                },
+                              )}
                               onChange={this.onChange}
-                              value={`${entry.id}:${role.id}`}
```
Comment (giuliaghisini) [inline src/components/manage/Sharing/Sharing.jsx:416]: why you have removed property 'value'? 
i think it's wrong

**Review by giuliaghisini — CHANGES_REQUESTED** [review]

File: src/components/manage/Sharing/Sharing.jsx (hunk 1 above)
Comment (Wagner3UB) [reply src/components/manage/Sharing/Sharing.jsx:416]: @giuliaghisini By mistake, I didn't want to delete.

**Review by giuliaghisini — APPROVED** [review]

File: src/components/manage/Sharing/Sharing.jsx [hunk 2]
```diff
@@ -83,6 +83,10 @@ const messages = defineMessages({
     id: 'Permissions have been updated successfully',
     defaultMessage: 'Permissions have been updated successfully',
   },
+  assignNewRoles: {
+    id: 'Assign the {entry} to the {role} role',
+    defaultMessage: 'Assign the {entry} to the {role} role',
```
Comment (davisagli) [inline src/components/manage/Sharing/Sharing.jsx:88]: This seems backwards in English; the checkbox assigns a role to a user or group. I think it should be:

```suggestion
    id: 'Assign the {role} role to {entry}',
    defaultMessage: 'Assign the {role} role to {entry}',
```

Comment (Wagner3UB) [reply src/components/manage/Sharing/Sharing.jsx]: Thanks @davisagli. I made the changes you suggested.

Comment (davisagli) [reply src/components/manage/Sharing/Sharing.jsx]: @Wagner3UB It's better, but in English we would say "Assign the Editor role to davisagli", not "Assign the Editor role to the davisagli"

**Review by davisagli — APPROVED** [review]

**Pull request merged**